### ISSUE TITLE
Categories for icons

### ISSUE THREAD

**Issue by My1** [winauth/winauth#345]
I think that some of the categories of icons should be renamed and/or reorganized.

this is just my opinion, feel free to disagree.

I think crypto is a bad name for bitcoing etc since even if btc is a cryptocurrency, crypto usually refers to cryptography in general e.g. encryption/decryption etc etc.
I'd rather make a category called "money/shopping" with paypal, amazon ebay, all the btc stuff. and so on.
also the software and social parts are a bit ambigous especially the software part. and why is ebay in social ebay is iirc just for selling/buying things, not a social network.
personally I would move everything in the software category that is below mac (in version 3.4.24) into a "web" category

**Comment by winauth** [comment]
Yes, definitely agree. I just threw the last batch of icons in within reorganising, but it needs it.
